mock-certify-plugin/pom.xml

```diff
@@ -73,7 +73,7 @@
         <dependency>
             <groupId>io.mosip.esignet</groupId>
             <artifactId>esignet-core</artifactId>
-            <version>1.4.0</version>
+            <version>1.4.1</version>
```

Comment:
Lets check with eSignet team, when the fix with serial number will be released ?

Reply:
According to Anusha, serial id is added to 1L for develop branch. It is staged for 1.5.0 release.